src/app/api/volunteer/auth/password-reset/validate/route.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  deletePasswordResetField,
+  getPasswordByToken,
+} from "@server/mongodb/actions/PasswordReset";
+import APIWrapper from "@server/utils/APIWrapper";
+
+export const GET = APIWrapper({
+  config: {
+    requireToken: true,
```

Comment:
This route should not require a token since the user will not be signed in for password reset.